bpf/lib/nodeport.h

```diff
@@ -2979,6 +2986,8 @@ static __always_inline int nodeport_lb4(struct __ctx_buff *ctx,
 #endif /* DSR_ENCAP_MODE */
 		ep_tail_call(ctx, CILIUM_CALL_IPV4_NODEPORT_DSR);
 	} else {
+		ctx_store_meta(ctx, CB_SRC_LABEL, src_sec_identity);
+		ctx_store_meta(ctx, CB_CLUSTER_ID_EGRESS, cluster_id);
```

Comment:
FWIW, I think it would be worth doing separate patches for the `cluster_id` and `src_sec_identity` change. They are handling different aspects - and while the `cluster_id` change is trivial, the `src_sec_identity` thing has more potential to need a revert.

Reply:
Sorry, I should have been clearer. I meant to extract *all* of the `cluster_id` related changes into a separate patch. Not just this single line.